.travis.yml

```diff
@@ -40,8 +40,8 @@ install:
   - travis_install_nightly 8.0 # only used if VERSION not set in env
 
 script:
-  - self_tests
+  - coverage run --rcfile=.coveragerc_mqt ./travis/self_tests
   - travis_run_tests 8.0  # only used if VERSION not set in env
```

Comment:
@dreispt Should I cover this line?

I need an absolute path, I can't use `$PATH` directly because of the `cp` in the setup

If I cover it, I need to change the line to :
```
- coverage run --rcfile=.coveragerc_mqt ./travis/travis_run_tests 8.0  # only used if VERSION not set in env
```

Currently, `travis_run_tests` is in `$HOME` not `$HOME/travis`, the coveralls won't work at associating the files like that.